README.md

```diff
@@ -1,288 +1,147 @@
   <ul>
-    <li><a href="https://reactjs.org/">React.js</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML">HTML</a></li>
   </ul>
 </details>
 
 <details>
-  <summary>Server</summary>
+<summary>Style</summary>
   <ul>
-    <li><a href="https://expressjs.com/">Express.js</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
   </ul>
 </details>
 
 <details>
-<summary>Database</summary>
+<summary>Layout</summary>
   <ul>
-    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Learn/CSS/CSS_layout/Flexbox">Flexbox</a></li>
   </ul>
 </details>
 
-<!-- Features -->
-
 ### Key Features <a name="key-features"></a>
 
-> Describe between 1-3 key features of the application.
-
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- LIVE DEMO -->
-
-## 🚀 Live Demo <a name="live-demo"></a>
+> Some useful features
 
-> Add a link to your deployed project.
+- **[Using Flexbox for the layout]**
+- **[Pure html]**
+- **[Pure css]**
 
-- [Live Demo Link](https://google.com)
 
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- GETTING STARTED -->
-
-## 💻 Getting Started <a name="getting-started"></a>
-
-> Describe how a new developer could make use of your project.
+## Getting Started
 
-To get a local copy up and running, follow these steps.
+To get a local copy up and running follow these simple example steps.
 
 ### Prerequisites
-
-In order to run this project you need:
-
-<!--
-Example command:
-
-```sh
- gem install rails
-```
- -->
+- Visual Studio Code must be installed on your computer | Download Visual Studio Code [VSCode](https://code.visualstudio.com/)
+- Git must be installed on your computer | Download [Git](https://git-scm.com/downloads)
 
 ### Setup
-
-Clone this repository to your desired folder:
-
-<!--
-Example commands:
-
-```sh
-  cd my-folder
-  git clone git@github.com:myaccount/my-project.git
-```
---->
+- Clone the repository
+- Open the project folder in Visual Studio Code
 
 ### Install
 
-Install this project with:
+Install this project using:
 
-<!--
-Example command:
-
-```sh
-  cd my-project
-  gem install
-```
---->
+- cd project/directory
+- Git clone [portfolio](https://github.com/jecode93/portfolio)
 
 ### Usage
 
 To run the project, execute the following command:
 
-<!--
-Example command:
+- Launch index.html file in your browser
 
-```sh
-  rails server
-```
---->
 
 ### Run tests
 
 To run tests, run the following command:
 
-<!--
-Example command:
-
-```sh
-  bin/rails test test/models/article_test.rb
-```
---->
+- Open the index.html file
+- Modify the name in the h1 tag or the description
+- Refresh you browser to see the changes
 
-### Deployment
 
-You can deploy this project using:
 
-<!--
-Example:
-
-```sh
-
-```
- -->
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] Please add this section back because except for `FAQ` rest of everything is required and you must have to add it because the template provided by *Microverse* is fully professional. You can write something about the web platforms which provide deployment services as information in this section.